crates/lovely-core/src/patch/pattern.rs

```diff
@@ -34,25 +34,77 @@ impl PatternPatch {
         if self.target != target {
             return false;
         }
+        if self.target.is_empty() {
+            return false;
+        }
```

Comment:
huh, is special handling needed if target is empty? that would be an empty string as filename which just wouldn't exist, I think

Reply:
Yeah, good point. Changed.